proto/errors.go

```diff
@@ -23,6 +23,11 @@ import (
 	"github.com/cockroachdb/cockroach/util/retry"
 )
 
+// ErrorUnexpectedlySet is the string we panic on when we assert
+// `ResponseHeader.Error == nil` before calling
+// `ResponseHeader.SetGoError()`.
+const ErrorUnexpectedlySet = "error is unexpectedly set"
```

Comment:
Yeah, I agree, but this thing is not long for this world; I think it's fine to leave it here for a few days until this transition is complete.